File: BECore/app/model/Degree.js
/// <summary>
/// Model EXTJS Degree.
/// </summary>
Ext.define('QLDT.model.Degree', {
    extend: 'Ext.data.Model',

    idProperty: 'DegreeID',
    idType: 'Guid',

    code: 'DegreeCode',
    name: 'DegreeName',

    fields: [
        { name: "DegreeID", type: "string" },
		{ name:"DegreeCode", type:"string" },
		{ name:"DegreeName", type:"string" },
		{ name:"DegreeShortName", type:"string" },
		{ name:"Description", type:"string" },
		{ name:"Inactive", type:"boolean" },
		{ name:"SortOrder", type:"int" }
     ]
});

File: MT.Model/Dictionary/Degree.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region Degree:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a Degree:EntityBase.
    /// </summary>
    public class Degree : EntityBase
    {
        [Key]
        public Guid DegreeID { get; set; }
        public string DegreeCode { get; set; }
        public string DegreeName { get; set; }
        public string DegreeShortName { get; set; }
        public string Description { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }



    }
    #endregion
}

